prompts/template_xml/corpus_generation_via_python_script.txt

```diff
@@ -0,0 +1,23 @@
+Hi! I'm a security engineer that is looking to improve the quality of my fuzzing harness.
+
+The goal is to look at how the corpus of the fuzzing harness can be improved.
+
+The project is {PROJECT_NAME} and the repository of the project is {PROJECT_REPOSITORY}.
+
+The fuzzer I'm working on is:
+<code>
+{HARNESS_SOURCE_CODE}
+</code>
+
+The target code of the harness is:
+<code>
+{TARGET_FUNCTION_SOURCE}
```

Comment:
Would this benefit from more context? e.g. relevant data type information? existing seeds in the repository?

Reply:
+1 to data type, e.g., `struct` detail if this is ready for functions with complex/project-specific data types.